components/datetime/src/format/mod.rs

```diff
@@ -2,8 +2,20 @@
 // called LICENSE at the top level of the ICU4X source tree
 // (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
 
+#[cfg(feature = "experimental")]
+use crate::neo_skeleton::FractionalSecondDigits;
+
 pub mod datetime;
 #[cfg(feature = "experimental")]
 pub mod neo;
 pub mod time_zone;
 pub mod zoned_datetime;
+
+/// Internal non-pattern runtime options passed to the formatter
+#[derive(Debug, Copy, Clone, Default)]
+pub(crate) struct FormattingOptions {
+    #[cfg(feature = "experimental")]
```

Comment:
question: this goes away with neo, yes?

Reply:
Since this is a new type I'd kinda prefer to avoid experimental-gating this temporarily